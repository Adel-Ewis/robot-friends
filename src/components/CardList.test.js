import { shallow } from 'enzyme';
import CardList from './CardList';
import React from 'react';

it('carList renders', () => {
  const mockRobots = [{ id: 1, name: 'solid snake', email: 'big_boss@fox.com' }];
  expect(shallow(<CardList robots={mockRobots} />)).toMatchSnapshot();
});
