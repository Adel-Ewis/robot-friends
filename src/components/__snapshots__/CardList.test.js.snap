// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`carList renders 1`] = `
<div>
  <Card
    email="big_boss@fox.com"
    id={1}
    key="0"
    name="solid snake"
  />
</div>
`;
